doc/test-framework/scorecard.md

```diff
@@ -324,8 +325,24 @@ Example of a valid JSON output:
 
 **NOTE:** The `ScorecardOutput.Log` field is only intended to be used to log the scorecard's output and the scorecard will ignore that field if a plugin provides it.
 
+## Running the scorecard with a deployed CSV
+
+The scorecard can be run using only a [Cluster Service Version (CSV)][olm-csv], providing a way to test cluster-ready and non-SDK operators.
+
+Running with a CSV alone requires both the `--csv-path=<CSV manifest path>` and `--olm-deployed` flags are set. The scorecard assumes your CSV and relevant CRD's have been deployed onto the cluster using the OLM when using `--olm-deployed`. This [document][olm-deploy-operator] walks through bundling your CSV and CRD's, deploying the OLM on minikube or [OKD][okd], and deploying your operator. Once these steps have been completed, run the scorecard with both `--csv-path=<CSV manifest path>` and `--olm-deployed` flags.
+
+A few notes:
+
+- As of now, using the scorecard with a CSV does not permit multiple CR manifests to be set through the CLI/config/CSV annotations. You will have to tear down your operator in the cluster, re-deploy, and re-run the scorecard for each CR being tested. In the future the scorecard will fully support testing multiple CR's without requiring users to teardown/standup each time.
```

Comment:
Weren't we planning on using the CR examples from the CSV in this mode? Maybe we haven't gotten to that yet?

/cc @AlexNPavel

Reply:
That functionality exists, but alternatively users should be allowed to use `--cr-manifest` if they choose. They're mutually exclusive options.

Reply:
My understanding and reading of these notes is:
1. When using `--olm-deployed` and `--csv-manifest`, only a single CR is supported. And that CR can be provided on the command line, via config file, or in the CSV via `metadata.annotations['alm-examples']`
2. None of these options support multiple CRs when used in combination with `--olm-deployed` and `--csv-manifest`.
3. Multiple CRs _can_ are supported when not used in combination with `--olm-deployed` and `--csv-manifest`
4. Regardless of the number of CRs and presence (or not) of `--olm-deployed` and `--csv-manifest`, these config mechanisms are mutually exclusive. For example, you can't use both `--cr-manifest` and `metadata.annotations['alm-examples']`.
5. In the future, we will update the scorecard to support multiple CRs in combination with `--olm-deployed` and `--csv-manifest`
6. In the future, we will enable CRs to be provided using multiple config mechanisms simultaneously.

Is that all true?

Reply:
All correct, except I hadn't thought about 6. The only reason that would be useful is if a user does not want to add another example to their CSV and is testing a new CR. This could be useful but complicates the logic a bit, because we'd have to check for duplicate CR's.

Reply:
:+1: Thanks. I don't think 6 is all that important. If someone starts asking for that, maybe we'd do it.

I took a quick look at the code to refresh my understanding. I could be missing something, but it looks like it's only the `--olm-deployed` flag that limits the number of CRs and that limit only applies to `metadata.annotations['alm-examples']`. If `--cr-manifest` is set and has multiple CRs, it looks like the scorecard will run all of them even with `--olm-deployed`.

If you agree, I think we should update these notes to make these clarifications.

Reply:
You are correct, and I intended to make a PR to fix that (and a few other) issue(s) but forgot. Just made one: #1565.